### PR TITLE
DEVDOCS-2215: Updated multisite section

### PR DESCRIPTION
**Pull request by ektravel**
# [DEVDOCS-2215](https://jira.bigcommerce.com/browse/DEVDOCS-2215)

## What changed?
* updated Multisite section
* removed Channel Name link (it just links installation )

### CODE REVIEW

File: docs/bigcommerce-for-wordpress/getting-started/install.md [hunk 1]
```diff
@@ -22,26 +21,25 @@
...
+To install BigCommerce for WordPress, you will need to [install and activate the plugin](https://wordpress.org/plugins/bigcommerce/) in WordPress and connect your BigCommerce account. If you do not already have an account, you will be prompted to create one.
 
 For a walkthrough on installing and activating the plugin, see [Using BigCommerce for Wordpress](https://support.bigcommerce.com/s/article/BigCommerce-for-WordPress#installation) (BigCommerce Help Center).
 
 ## Setting your channel name
 
-Once connected, you will set up a new [Channel Name](https://support.bigcommerce.com/s/article/BigCommerce-for-WordPress#channel). This name will be used to identify this specific WordPress instance in BigCommerce when checking orders or listing products.
+Once connected, you will set up a new **Channel Name**. This name will be used to identify this specific WordPress instance in BigCommerce when checking orders or listing products.
```
Comment (traci707) [inline docs/bigcommerce-for-wordpress/getting-started/install.md:30]:
```suggestion
Once connected, you will set up a new **Channel Name**. This name identifies this specific WordPress instance in BigCommerce when checking orders or listing products.

```

**Review by traci707 — CHANGES_REQUESTED** [review]

**Review by uwikaiddi — CHANGES_REQUESTED** [review]
@ektravel It looks like the same sections are repeated here. Can you confirm which version is correct and remove the other?

**Comment by ektravel** [comment]
@uwikaiddi This PR contains changes made to **Multisite Instructions** which were not included on PR 673.

File: docs/bigcommerce-for-wordpress/getting-started/install.md [hunk 2]
```diff
@@ -1,3 +1,50 @@
+
+<div><h3 class="sub-docs-type" id="bigcommerce-for-wordpress">BigCommerce for Wordpress</h3>
+
+# Install
```
Comment (uwikaiddi) [inline docs/bigcommerce-for-wordpress/getting-started/install.md:4]: @ektravel, this section all the way down to Installation is a repeat.

File: docs/bigcommerce-for-wordpress/getting-started/install.md [hunk 3]
```diff
@@ -1,3 +1,50 @@
...
+
+</div>
+</div>
+</div>
+
+After you have created your new channel name, the initial product sync will begin, and you can begin configuring your [plugin settings](https://support.bigcommerce.com/s/article/BigCommerce-for-WordPress-Plugin-Settings).
+=======
 <div><h3 class="sub-docs-type" id="bigcommerce-for-wordpress">BigCommerce for Wordpress</h3>
 
 # Install
```
Comment (uwikaiddi) [inline docs/bigcommerce-for-wordpress/getting-started/install.md:50]: This section is repeated.

**Review by uwikaiddi — CHANGES_REQUESTED** [review]

**Pull request closed without merging**